lsdcGui.py

```diff
@@ -3061,7 +3061,7 @@ def processSampMove(self,posRBV,motID):
         self.vecLine.setLine(self.vectorStart["graphicsitem"].x()+self.vectorStart["centerCursorX"]+self.centerMarkerCharOffsetX,self.vectorStart["graphicsitem"].y()+self.vectorStart["centerCursorY"]+self.centerMarkerCharOffsetY,self.vectorEnd["graphicsitem"].x()+self.vectorStart["centerCursorX"]+self.centerMarkerCharOffsetX,self.vectorEnd["graphicsitem"].y()+self.vectorStart["centerCursorY"]+self.centerMarkerCharOffsetY)
 
     def queueEnScanCB(self):
-      self.protoComboBox.setCurrentIndex(self.protoComboBox.findText(str("eScan")))      
+      self.protoComboBox.setCurrentText("eScan")      
```

Comment:
You are correct, it is unusual. EScan is removed from the combo box list while on the main screen, so it can not be found with the "findText" when you queue from the e-scan tab. It does not appear on the list, instead the combo box on the main screen shows a blank when you switch back. 

Overall it prevents you from queueing e-scans from the main tab and automatically applies it as the selected type when on the e-scan tab.